mindsdb/integrations/handlers/autosklearn_handler/autosklearn_handler.py

```diff
@@ -0,0 +1,47 @@
+from typing import Optional, Dict
+
+import dill
+import pandas as pd
+import autosklearn.classification as automl_classification
+import autosklearn.regression as automl_regression
+
+from .config import ClassificationConfig, RegressionConfig
+
+from mindsdb.integrations.libs.base import BaseMLEngine
+
+
+class AutoSklearnHandler(BaseMLEngine):
+    """
+    Integration with the Auto-Sklearn ML library.
+    """
+
+    name = 'autosklearn'
+
+    def create(self, target: str, df: Optional[pd.DataFrame] = None, args: Optional[dict] = None) -> None:
+        config_args = {key: val for key, val in args['using'].items() if key != 'task'}
+
+        if args['using']['task'] == 'classification':
```

Comment:
Perhaps out of scope for this PR (I'll let you decide), but it would be pretty cool if we used `type_infer` here to automatically figure out if it's a regression or classification problem (or something else that is not supported!).

Reply:
Love this. It's done!